client/network/src/protocol/schema/api.v1.proto

```diff
@@ -51,5 +51,8 @@ message BlockData {
 	bytes message_queue = 5; // optional
 	// Justification if requested.
 	bytes justification = 6; // optional
+	// True if justification should be treated as present be empty.
```

Comment:
Comment does not make sense to me. Did you meant:

```suggestion
	// True if justification should be treated as empty.
```

Reply:
That was meant to be:
```suggestion
	// True if justification should be treated as present but empty.
```